ui/components/multichain/nft-item/nft-item.js

```diff
@@ -75,6 +76,7 @@ export const NftItem = ({
             name={name}
             tokenId={tokenId}
             clickable={clickable}
+            isIpfsEnabled={isIpfsEnabled}
```

Comment:
In the parent of this component `nft-items.js` or `nft-details.js`, I have added a check to verify if the image is based on IPFS data and get the src based on that, we need ipfsEnabled check because we are showing NftDefaultImage only if the ipfs is toggled off in that case isIpfsEnabled is true.

Reply:
My apologies, I was wrong here. With ipfsEnabled check, we were missing on if ipfs was Enabled and the image was stored on web2. Fixed it here https://github.com/MetaMask/metamask-extension/pull/20172/commits/df1ccda81462ae23a0229ec67e5f26f81a161771